app/graphql/mutations/user/add_watchlist_movie.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Mutations
+  module User
+    class AddWatchlistMovie < AuthBase
+      type Types::UserAddWatchlistMovieType
```

Comment:
please return `type Types::MovieType`
You should create specific input types for mutation, but not the object types (that are returned to the client)